Misc/NEWS.d/next/C API/2020-10-12-20-13-58.bpo-42015.X4H2_V.rst

```diff
@@ -0,0 +1,5 @@
+Order of dereferencing for `PyCFunction` deallocation changed to make sure
```

Comment:
It describes implementation details, how the bug was fixed. It is better to describe what bug was fixed, from the point of view of the user or author of third-party extension.

Something like (improve my poor language) "Fixed crash in deallocating method objects with dynamically allocated PyModuleDef."

Reply:
Right, yes. Something like this, then, perhaps? The link to the `self` argument might be important for users to know whether they had been affected or not, I reckoned?